### PR TITLE
fix(v4): update examples starting with "t" to migrate new router provider

### PR DESCRIPTION
**Pull request by salihozdemir**
Update examples starting with "t" to migrate v4.

### CODE REVIEW

File: examples/template-antd/src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
-            legacyRouterProvider={routerProvider}
-        />
+        <BrowserRouter>
+            <Refine
+                routerProvider={routerProvider}
+                dataProvider={dataProvider("https://api.fake-rest.refine.dev")}
+                notificationProvider={notificationProvider}
+            >
+                <Routes>
+                    <Route index element={<WelcomePage />} />
```
Comment (alicanerdurmaz) [inline examples/template-antd/src/App.tsx:18]: I believe we should add this 

```
<Route path="*" element={<ErrorComponent />} />
```

or 

```
<Route path="*" element={<Navigate to="/" />} />
```

File: examples/template-chakra-ui/src/App.tsx [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+            <ChakraProvider theme={refineTheme}>
+                <Refine
+                    routerProvider={routerProvider}
+                    dataProvider={dataProvider(
+                        "https://api.fake-rest.refine.dev",
+                    )}
+                    notificationProvider={notificationProvider}
+                >
+                    <Routes>
+                        <Route index element={<WelcomePage />} />
```
Comment (alicanerdurmaz) [inline examples/template-chakra-ui/src/App.tsx:24]: I believe we should add this 

```
<Route path="*" element={<ErrorComponent />} />
```

or 

```
<Route path="*" element={<Navigate to="/" />} />
```

File: examples/template-mantine/src/App.tsx [hunk 3]
```diff
@@ -1,37 +1,39 @@
...
+                    <Refine
+                        routerProvider={routerProvider}
+                        dataProvider={dataProvider(
+                            "https://api.fake-rest.refine.dev",
+                        )}
+                        notificationProvider={notificationProvider}
+                    >
+                        <Routes>
+                            <Route index element={<WelcomePage />} />
+                        </Routes>
```
Comment (alicanerdurmaz) [inline examples/template-mantine/src/App.tsx:32]: I believe we should add this 

```
<Route path="*" element={<ErrorComponent />} />
```

or 

```
<Route path="*" element={<Navigate to="/" />} />
```

File: examples/template-mui/src/App.tsx [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
+                <RefineSnackbarProvider>
+                    <Refine
+                        routerProvider={routerProvider}
+                        dataProvider={dataProvider(
+                            "https://api.fake-rest.refine.dev",
+                        )}
+                        notificationProvider={notificationProvider}
+                    >
+                        <Routes>
+                            <Route index element={<WelcomePage />} />
```
Comment (alicanerdurmaz) [inline examples/template-mui/src/App.tsx:31]: I believe we should add this 

```
<Route path="*" element={<ErrorComponent />} />
```

or 

```
<Route path="*" element={<Navigate to="/" />} />
```

**Review by alicanerdurmaz — DISMISSED** [review]

**Review by BatuhanW — APPROVED** [review]

**Pull request merged**